examples/setall.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+
+import sys
+from sys import argv
+import blinkt
+
+if len(argv)<2 or len(argv)>5:
+    sys.stderr.write( "Syntax: {0} <red> <green> <blue> [brightness]]\n".format(argv[0]) )
+    exit(1)
+
+red    = int(argv[1])   if len(argv)>3 else 0
```

Comment:
With the updated check above argv should *always* contain at least the red, green and blue values so you can drop the checks on these lines and simplify to:

```python
red = int(argv[1])
green = int(argv[2])
blue = int(argv[3])
```

Reply:
Done